modules/nf-core/isoseq/cluster/tests/main.nf.test

```diff
@@ -0,0 +1,46 @@
+nextflow_process {
+
+    name "Test Process ISOSEQ_CLUSTER"
+    script "../main.nf"
+    config "./nextflow.config"
+    process "ISOSEQ_CLUSTER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "isoseq"
+    tag "isoseq/cluster"
+
+    test("Main test") {
```

Comment:
Can you change the test to something more meaningful (Including data input and type) in this case probably
```suggestion
    test("human - pacbio - bam") {
```

Reply:
Done